onebusaway-android/src/main/java/org/onebusaway/android/map/bike/BikeStationLoader.java

```diff
@@ -0,0 +1,108 @@
+package org.onebusaway.android.map.bike;
```

Comment:
Could you please add Apache license header to the top of each new class?

If any code is a copy/paste/edit from other files in the project, the names of the people and orgs in the Apache license header of that file should be included here, along with your name, and the date range should be the start date in that file header with end date 2017.  If all code is entirely yours in the new file, then your name would be the only one listed and 2017 would be only date listed.

So, if all code is yours, it would look like this:

~~~
/*
 * Copyright (C) 2017 Rodrigo Carvalho (your.email@emailprovider.org)
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
~~~

If there was copy pasted code from another file, it would look something like this:

~~~
/*
 * Copyright (C) 2013-2017 University of South Florida (sjbarbeau@gmail.com), Rodrigo Carvalho (your.email@emailprovider.org)
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
~~~